esp8266-fastled-webserver/Map.cpp

```diff
@@ -68,7 +68,7 @@
   const uint8_t  coordsY[NUM_PIXELS]             { 126, 120, 109, 96, 82, 69, 57, 49, 45, 45, 50, 59, 74, 92, 114, 138, 163, 188, 211, 231, 255, 248, 235, 218, 198, 175, 152, 129, 107, 89, 74, 63, 57, 56, 59, 66, 76, 88, 102, 116, 103, 88, 77, 71, 68, 70, 77, 88, 103, 121, 141, 163, 184, 205, 222, 236, 245, 249, 247, 208, 224, 235, 241, 240, 234, 223, 209, 191, 172, 152, 132, 115, 101, 90, 84, 82, 86, 95, 114, 107, 98, 98, 103, 112, 126, 142, 159, 177, 195, 210, 222, 230, 233, 230, 223, 209, 191, 168, 142, 98, 125, 151, 174, 194, 209, 219, 223, 223, 218, 208, 195, 180, 164, 148, 134, 122, 114, 112, 123, 128, 138, 151, 165, 180, 193, 203, 211, 214, 212, 206, 194, 178, 158, 134, 109, 83, 58, 35, 11, 28, 48, 71, 95, 120, 142, 163, 179, 192, 200, 203, 202, 196, 187, 175, 162, 148, 136, 133, 152, 166, 177, 186, 190, 191, 187, 178, 165, 148, 128, 107, 84, 62, 41, 24, 11, 2, 0, 28, 16, 9, 8, 13, 23, 37, 55, 75, 96, 116, 135, 151, 164, 173, 177, 177, 172, 162, 146, 153, 161, 163, 160, 152, 139, 124, 106, 87, 69, 51, 36, 25, 18, 16, 20, 29, 44, 64, 133, 106, 81, 60, 44, 32, 26, 25, 29, 38, 50, 65, 82, 99, 115, 129, 140, 147, 148, 138, 134, 131, 122, 110, 95, 80, 65, 52, 42, 36, 34, 37, 45, 59, 77, 98, 123, 149, 176, 202 };
   const uint8_t  angles[NUM_PIXELS]              { 0, 247, 238, 229, 220, 211, 203, 194, 185, 176, 167, 159, 150, 141, 132, 123, 115, 106, 97, 88, 65, 74, 83, 92, 100, 109, 118, 127, 136, 144, 153, 162, 171, 180, 188, 197, 206, 215, 224, 232, 209, 201, 192, 183, 174, 165, 157, 148, 139, 130, 121, 113, 104, 95, 86, 77, 69, 60, 51, 28, 37, 46, 54, 63, 72, 81, 90, 98, 107, 116, 125, 134, 142, 151, 160, 169, 178, 186, 195, 172, 163, 155, 146, 137, 128, 119, 111, 102, 93, 84, 75, 67, 58, 49, 40, 31, 23, 14, 5, 246, 255, 8, 17, 26, 35, 44, 52, 61, 70, 79, 88, 96, 105, 114, 123, 132, 140, 149, 135, 126, 117, 108, 100, 91, 82, 73, 64, 56, 47, 38, 29, 20, 12, 3, 250, 241, 232, 223, 209, 218, 227, 235, 244, 253, 6, 15, 24, 33, 41, 50, 59, 68, 77, 85, 94, 103, 112, 98, 89, 80, 71, 62, 54, 45, 36, 27, 18, 10, 1, 247, 239, 230, 221, 212, 203, 195, 186, 163, 172, 180, 189, 198, 207, 216, 224, 233, 242, 251, 4, 13, 22, 31, 39, 48, 57, 66, 75, 52, 43, 34, 25, 16, 8, 254, 245, 237, 228, 219, 210, 201, 193, 184, 175, 166, 157, 149, 126, 134, 143, 152, 161, 170, 178, 187, 196, 205, 214, 222, 231, 240, 249, 2, 11, 20, 28, 37, 14, 5, 252, 243, 235, 226, 217, 208, 199, 191, 182, 173, 164, 155, 147, 138, 129, 120, 111, 103 };
-  static const uint8_t (&radiusProxy)[NUM_PIXELS] = physicalToFibonacci;
+  const uint8_t (&radiusProxy)[NUM_PIXELS] = physicalToFibonacci;
```

Comment:
Was the removal of `static` here intentional?

Reply:
static simply ensures the value doesn't leak into other compilation units.  Since it's not a functional difference, I'll approve.  Adjust or not as you see fit.

Reply:
I had to remove `static` or I got an error here: https://github.com/jasoncoon/esp8266-fastled-webserver/pull/229/files#diff-add2ca75aa27ae4932f01dfa0cc7a2b9f67589094f6e3eaa7fff67c7f200b186R232

```
esp8266-fastled-webserver/Map.cpp:71:49: error: 'radiusProxy' was declared 'extern' and later 'static' [-fpermissive]
   static const uint8_t (&radiusProxy)[NUM_PIXELS] = physicalToFibonacci;
                                                 ^
In file included from esp8266-fastled-webserver/Map.cpp:1:0:
esp8266-fastled-webserver/common.h:232:26: error: previous declaration of 'radiusProxy' [-fpermissive]
   extern const uint8_t (&radiusProxy)[NUM_PIXELS];  // needed in noise.cpp
```

If I tried updating that to `extern static const` I get `more than one storage class may not be specified`.

I freely admit my C ignorance here. :)

Reply:
Ah... I missed that you had modified `noise.cpp` to use the radius proxy rather than directly use the radii.  OK, this now all makes sense.  👍 Good to go!